.github/workflows/initialize.yml

```diff
@@ -147,20 +147,15 @@ jobs:
           git config --global user.email "noreply@github.com"
           git config --global user.name "GitHub"
 
+          git checkout -b init-root-targets
           # Commit the REPO changes
           git add ${REPO}
           git commit -s -m "Add staged repository metadata"
+          git push origin HEAD
```

Comment:
@kommendorkapten Is HEAD correct here? Or is this supposed to be a branch?

Reply:
@haydentherapper `HEAD` is correct here, as it refers to the active branch `init-root-targets`, which is the one we will create the PR from. Does this cause any problems?

Reply:
No issue, just wanted to double check.